include/toolchain/gcc.h

```diff
@@ -490,6 +502,7 @@ do {                                                                    \
 	__asm__(".globl\t" #name                    \
 		"\n\t.equ\t" #name "," #value       \
```

Comment:
What happens if we remove the type directive w/gcc?

Reply:
That's a really good question. TBH, I'm not sure, but I feel like it could have negative affects for armv*m because of thumb mode. I could be wrong.

Reply:
Oh wait, actually, in this case most likely nothing, since it's native posix.